### PR TITLE
task(output): Adding console writer class and concrete implementations

### PR DESCRIPTION
**Pull request by scalvert**

### CODE REVIEW

File: src/utils/console-writer.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    this.text(COLORS.heading(`${this.indent(2)}${heading}`));
+    this.divider();
+    this.line();
+  }
+
+  pairs(data: { [key: string]: string }) {
+    console.log(
+      columnify(data, {
+        dataTransform: (data: string) => {
+          return this.indent(2) + data;
```
Comment (suchitadoshi1987) [inline src/utils/console-writer.ts]: nit: use template literal for concatenation

Comment (scalvert) [reply src/utils/console-writer.ts]: I mean, this is the difference of `${this.indent(2)}${data}` vs. the above. Does this really bother you?

Comment (suchitadoshi1987) [reply src/utils/console-writer.ts]: It's more about consistency than the difference above that I prefer :)
ref: https://github.com/scalvert/ember-cli-checkup/pull/50/files/8d3d160f44b8b90da09f9400634eb26d9e3fdbbc#diff-49e69fc097ef947f6d44ef1f35d72075R18

Comment (scalvert) [reply src/utils/console-writer.ts]: Done!

**Pull request merged**